core/src/main/scala/kafka/coordinator/GroupMetadata.scala

```diff
@@ -201,6 +215,19 @@ private[coordinator] class GroupMetadata(val groupId: String, val protocolType:
     members.map{ case (memberId, memberMetadata) => (memberId, memberMetadata.metadata(protocol))}.toMap
   }
 
+  def summary: GroupSummary = {
+    if (is(Stable)) {
+      val members = this.members.values.map{ member => member.summary(protocol) }.toList
+      GroupSummary(state.toString, protocolType, protocol, members)
+    } else {
+      GroupSummary(state.toString, protocolType, GroupCoordinator.NoProtocol, GroupCoordinator.NoMembers)
```

Comment:
I think it maybe useful to also return the current members with empty metadata / assignment in this case?